reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -76,8 +82,8 @@ def __init__(
             raise ValueError("Threshold cannot be < 1")
 
         # column for mapping user / item ids to internal indices
-        self.col_item_id = sar.INDEXED_ITEMS
-        self.col_user_id = sar.INDEXED_USERS
+        self.col_item_id = '_indexed_items'
```

Comment:
Black has not been passed here. In relation to this, we should consider to add black in the precommit hooks like the NLP and CV repos

Reply:
ah, ok. sounds good